metadata-models/src/main/pegasus/com/linkedin/datajob/DataJobInputOutput.pdl

```diff
@@ -60,4 +61,26 @@ record DataJobInputOutput {
     }
   }
   inputDatajobs: optional array[DataJobUrn] 
+
+  @Searchable = {
+    "/*": {
+      "fieldName": "inputFields",
+      "fieldType": "URN",
+      "numValuesFieldName": "numInputFields",
+      "queryByDefault": false
+    }
+  }
+  inputDatasetFields: optional array[Urn]
+
+  @Searchable = {
+    "/*": {
+      "fieldName": "outputFields",
+      "fieldType": "URN",
+      "numValuesFieldName": "numOutputFields",
+      "queryByDefault": false
+    }
+  }
+  outputDatasetFields: optional array[Urn]
+
+  fieldsLineages: optional FieldsLineages
```

Comment:
/cc @jjoyce0510 suggestions for a different name here?
maybe `fieldLevelLineage: optional FieldLevelLineage`